engine/src/main/resources/assets/i18n/menu_cs.lang

```diff
@@ -86,7 +86,7 @@
     "drop-item": "Zahodit předmět",
     "edit-server": "Upravit",
     "edit-server-title": "Přidat/Upravit server",
-    "enter-username-message": "Ahoj a vítej v muliplayeru Terasology!\nChceš použít vlastní jméno?",
+//    "enter-username-message": "Ahoj a vítej v muliplayeru Terasology!\nChceš použít vlastní jméno?",
```

Comment:
Not good to have comments inside JSON let alone commented out strings - better to leave an associated TODO in a .java that relates or submit an issue to figure out what we need to do :-)